tests/unit_tests/test_routes.py

```diff
@@ -93,7 +93,7 @@ def __fake_get_dynamo_db(self):
     def test_checker_ok(self):
         # checker
         self.setUp()
```

Comment:
Not sure if this call is needed at the beginning of every test. I thought, that the `setUp` is called by the test runner automatically, if it exists. But then, I am also not sure, if it does any harm to call it anyways ;-)

Reply:
good catch this needs to be removed otherwise the setUp() method is called twice, once by the framework and once inside the method.